nautobot_circuit_maintenance/handle_notifications/handler.py

```diff
@@ -23,19 +24,19 @@
 
 
 def create_circuit_maintenance(
-    logger: Job, maintenance_id: str, parsed_notification: ParsedNotification, provider: Provider
+    logger: Job, maintenance_id: str, parser_maintenance, provider: Provider
```

Comment:
What's the type of `parser_maintenance` now?

Same question on each function that has this as an argument below.

Reply:
it's `Maintenance` from the parser library (I added a TODO comment in some place), so I'm waiting for the official release to update